server.js

```diff
@@ -1,64 +0,0 @@
-const express = require('express')
```

Comment:
This whole file was moved to `./src/server.js` and refactored. See that file below.